src/Stryker.ts

```diff
@@ -102,6 +104,13 @@ export default class Stryker {
     });
   }
 
+  private freezeConfig() {
+    freezeRecursively(this.config);
+    if (log.isDebugEnabled()) {
```

Comment:
Is this check added for performance reasons as you don't want to spend time stringifying the config the user does not even see the log message?